adapters/InMobi/InMobiAdapter/GADMAdapterInMobiRewardedAd.h

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
```

Comment:
Remove, the same applies to all the other places.

Reply:
done